mlir/utils/jenkins/Jenkinsfile

```diff
@@ -812,7 +814,7 @@ pipeline {
                 axes {
                     axis {
                         name 'CHIP'
-                        values 'gfx908', 'gfx90a', 'gfx1030', 'gfx1101'
+                        values 'gfx908', 'gfx90a', 'gfx1030', 'gfx1100', 'gfx1201'
```

Comment:
can you trigger nightly and weekly tuning runs manually on this branch and see if it is tuning and generating perf reports  ?